rest-service/manager_rest/resources.py

```diff
@@ -1383,7 +1384,7 @@ def post(self):
             get_blueprints_manager().update_provider_context(update, context)
```

Comment:
in fact you just want to check _update_import_resolver_in_app, that is called as part of update_provider_context. seems to me like it should be extracted from it and called separately.